kiliautoml/utils/pytorchvision/image_classification.py

```diff
@@ -145,30 +145,28 @@ def initialize_model_img_class(model_name: ModelNameT, class_names):
 def predict_probabilities(
     loader: torch_Data.DataLoader,  # type: ignore
     model,
-    verbose=0,
+    verbose: VerboseLevelT,
 ) -> List[float]:
     """
     Method to compute the probabilities for all classes for the assets in the holdout set
     """
     # Switch to evaluate mode.
+    _ = verbose
```

Comment:
strange

Reply:
verbose is not used in this function but is declared in the base class. Since now why use logging, we no longer verbose argument in a bunch of places. Otherwise, flake8 would not be happy

Reply:
that's what I thought, but maybe there is no need for this argument in the base class and you can add it only to the classes that need it